package.json

```diff
@@ -46,6 +47,7 @@
     "sanitize-filename": "^1.6.1",
     "sinon": "^13.0.1",
     "sinon-chai": "^3.7.0",
+    "ts-node": "^10.9.1",
```

Comment:
Adding the `ts-node` dependency simplifies the running of the included script by removing the need to transpile the Typescript code into Javascript.